model_compression_toolkit/qat/keras/qat_model_builder.py

```diff
@@ -92,40 +102,3 @@ def _quantize_node_activations(self,
 
         """
         return node.final_activation_quantization_cfg.quantize_node_output(input_tensors)
-
-    def build_model(self) -> Tuple[Model, UserInformation]:
```

Comment:
if I'm not mistaken, this part is still required.. it should contain:
qat_model = tf.keras.models.clone_model(model,
                                                input_tensors=None,
                                                clone_function=qat_wrapper)